### ISSUE TITLE
[Site Request] batcave.biz

### ISSUE THREAD

**Issue by noorhakim** [manga-download/hakuneko#7329]
**Name of the website**
batcave.biz

**Website urls (examples below)**
- Site: https://batcave.biz/
- Manga List: https://batcave.biz/comix/
- Manga example: https://batcave.biz/30310-beneath-the-trees-where-nobody-sees-2023.html
- Chapter online viewer example: https://batcave.biz/30310-beneath-the-trees-where-nobody-sees-2023.html

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

**Comment by MikeZeDev** [comment]
Ahaha i love the website name 🦇 

But 
![image](https://github.com/user-attachments/assets/e78463e5-e724-49a6-bc09-a232b8f3d1a5)

on every comic.. Retarded website want mobile user agent 🤦

**Comment by noorhakim** [comment]
It's also happened to me when I'm using vless connection. But, it's working fine with my phone data.

**Comment by MikeZeDev** [comment]
Is login mandatory now?

**Comment by noorhakim** [comment]
> Is login mandatory now?

No. I tried without login and it's fine. Are you working on it? I thought you not because mobile user agent problem

**Comment by MikeZeDev** [comment]
As long as i click on a comic its asking me to login

**Comment by noorhakim** [comment]
Ah. I supposed you are using web browser on pc/laptop? I'm also facing same problem if using it. But, when using mobile phone it's working fine

**Comment by 27136773** [comment]
I can reach the website just fine from MacOS with Vivaldi
Login seems required for access from regions such as France (Indonesia worked for me)

**Comment by MikeZeDev** [comment]
#No, Website has changed. its when you want to access mangas that it ask for login if no mobile agent.
When i first answered, we couldnt access the website at all.

**Comment by xx88xx** [comment]
Did this ever get fixed? Super useful source for comics

**Comment by xx88xx** [comment]
@manga-download-owner - why was my comment deleted?

**Comment by MikeZeDev** [comment]
> [@manga-download-owner](https://github.com/manga-download-owner) - why was my comment deleted?

Github is not for chitchatting especially asking for ETA but to provide useful conversations about issues like : explanations about bugs, how to solve them, how to mitigate them, what does users / dev must do, etc..., etc.. As for all open source project, its done when someone want to do it and asking for it is certainly not a good idea :)

When you ask for ETA it just create noise for the dev that check his notifications, and deleting the message is less time consuming than taking the time to answer properly, which sadly we cant always afford to do 😔 .

TLDR : dont ask for ETA ;)

**Comment by xx88xx** [comment]
Did I ask for an ETA or did I just ask if it’s just possible? I thought the
later

**Comment by manga-download-owner** [comment]
> [@manga-download-owner](https://github.com/manga-download-owner) - why was my comment deleted?

Because your question
<details>
<summary>[show]</summary>

> Did this ever get fixed? Super useful source for comics
</details>

- didn't add any value to solve the issue
- was expressed very intimidating in a way trying to put pressure on the contributors and attempting to raise the priority

Since a question about _"why a comment was deleted"_ itself adds no value to solve the issue, it should be deleted as well.
However i will leave this explanation as reference for future incidents.

**Comment by MikeZeDev** [comment]
Website is not even supported and people posts here to ask for help because they got http errors? o_o. Locking.
